extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -138,6 +138,21 @@ export class ProjectsController {
 		deployDatabaseDialog.openDialog();
 	}
 
+	public async schemaCompare(treeNode: BaseProjectTreeItem): Promise<void> {
+		// check if schema compare extension is installed
+		if (this.apiWrapper.getExtension(constants.schemaCompareExtensionId)) {
+			// build project
+			await this.buildProject(treeNode);
+
+			// start schema compare with the dacpac produced from build
+			const project = this.getProjectContextFromTreeNode(treeNode);
+			const dacpacPath = path.join(project.projectFolderPath, 'bin', 'Debug', `${project.projectFileName}.dacpac`);
+			this.apiWrapper.executeCommand('schemaCompare.start', dacpacPath);
+		} else {
+			this.apiWrapper.showErrorMessage(constants.schemaCompareNotInstalled);
```

Comment:
not necessary for this PR but can we have a "do you want to install - yes/no" and opening the extn. viewlet with SC searched?. Is the second part (opening Extn view with SC) possible?

Reply:
I like this idea!  Help reduce friction for using the feature

Reply:
opened issue to track this #10415